src/VisualStudio/TestUtilities2/CodeModel/CodeModelTestState.vb

```diff
@@ -23,10 +23,12 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.CodeModel
         )
 
             If workspace Is Nothing Then
+                GC.SuppressFinalize(Me)
```

Comment:
Why?

Reply:
Because someone wrote a finalizer.

Reply:
Can you comment in the code?

Reply:
This change is now removed from this pull request.

I would not comment in the code, but I would be happy to remove the finalizer.